01/app.js

```diff
@@ -0,0 +1,21 @@
+const form = document.querySelector("form");
+const getFirstName = document.querySelector("input[name='firstName']");
+const getLastName = document.querySelector("input[name='lastName']");
```

Comment:
Raczej początek nazwy zmiennej tj. `get...` sugeruje, że to funkcja (pobierz), a to jest element HTML więc trochę to nie pasuje.